superset/connectors/sqla/models.py

```diff
@@ -1410,7 +1410,7 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
                         col=selected, template_processor=template_processor
                     )
                 groupby_all_columns[outer.name] = outer
-                if not series_column_names or outer.name in series_column_names:
+                if outer.name in series_column_names:
```

Comment:
Can you update this line for the `helpers.py` as well?

https://github.com/preset-io/superset/blob/cf02934b21ea608557ca600c28efc680647ef130/superset/models/helpers.py#L1518

Reply:
I think this should work - legacy timeseries charts will have set `is_timeseries` to `True`, and new charts will no longer be using it:
```suggestion
                if (
                    is_timeseries and not series_column_names
                ) or outer.name in series_column_names:
```

Reply:
@villebro That makes sense to me, thanks!  I can do some double-checking with the legacy timeseries charts – which ones are you talking about?

Reply:
The legacy time series line, bar etc charts. Essentially the ones with `is_timeseries` set to `True` and that are served by `viz.py`

Reply:
Ok great, I'll update the PR ASAP!